internal/reader/home_chain.go

```diff
@@ -16,7 +16,10 @@ import (
 	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
 )
 
-//go:generate mockery --name HomeChain --output ./mocks/ --case underscore
```

Comment:
q: shouldn't we keep this? Are the mocks generated somewhere else?

Reply:
removed the obselete comment

Reply:
there was a comment added to use mockery command. that was not supposed to be here.it must be in readme doc.
so deleted that from code @dimkouv

Reply:
I think we should keep the go:generate here.
Created the ticket CCIP-2901 that @winder will work on.

Reply:
retained the comment back